### DISCUSSION TITLE
What can I do about stolen code?

### DISCUSSION THREAD

**Discussion by tfinnm** [community/community#22245] [New to GitHub]
<p>I had a cloud service that I built uploaded to a private repo so that I could get a second opinion from some other developers I knew. The code itself was proprietary, as in, unlike my other projects, I have never given anyone a license to use/copy/modify/etc any of the code, but a few months ago I found my repo reuploaded with no changes except that the person who reuploaded it had added their own copyright notices to it… normally I would be fine with this, especially in my projects where I include a license that allows this, and if they had done this without including their own copyright notices I doubt I would’ve cared, but how they did this really annoys me. Back when I found it I submitted a pull request to their repo which just removed the infringing code, but after a few months, during which time I also tried to contact the author on other platforms, they’ve just ignored the pull request, so what else can I do?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can ask Github to remove the code. There’s a lot of legal stuff involved, so I’m just going to point at the relevant documentation:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/submitting-content-removal-requests" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/submitting-content-removal-requests" target="_blank" rel="noopener">Submitting content removal requests - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
